interface/main/tabs/templates/patient_data_template.php

```diff
@@ -55,8 +53,8 @@
         <span class="patientDataColumn">
         <!-- ko if: patient -->
         <!-- ko with: patient -->
-            <a class="css_button_small" data-bind="click: clickNewEncounter" href="#" title="<?php echo xlt("New Encounter");?>">
-                <i style="font-size:150%;" class="fa fa-plus"></i>
+            <a class="btn btn-xs btn-link" data-bind="click: clickNewEncounter" href="#" title="<?php echo xlt("New Encounter");?>">
```

Comment:
I am not sure how I missed this when this was brought in, but this should be xla() since within html attribute.